src/react/get-params.js

```diff
@@ -33,7 +33,17 @@ function getParams(obj = {}) {
     }
   });
   ['navigation', 'pagination', 'scrollbar'].forEach((key) => {
-    if (params[key] === true || params[key] === false) params[key] = {};
```

Comment:
Why do you need to specify default value?
Wouldnt it be enough to change it to:
```js
if (params[key] === true) params[key] = {};
```

Reply:
I wondered if I needed to convert it to an object, so I set the default value.
That change seems to be enough, so I will change it.